fabtests/functional/unexpected_msg.c

```diff
@@ -199,6 +200,96 @@ static int run_test_loop(void)
 	return ret;
 }
 
+static int exchange_unexp_addr(void)
+{
+	char temp[FT_MAX_CTRL_MSG];
+	size_t addrlen;
+	int ret;
+
+	if (opts.dst_addr) {
+		ret = ft_av_insert(av, fi->dest_addr, 1, &remote_fi_addr, 0,
```

Comment:
This line caused the segfault for efa provider. Efa provider currently only support out-of-band address exchange in ft_init_av, where the address to be inserted is the ep name sent from peer via sockets.

What is the fi->dest_addr here in this case?